DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/Index.cshtml

```diff
@@ -27,19 +27,56 @@
   <div class="nhsuk-grid-column-three-quarters">
     <div class="nhsuk-grid-row">
       <div class="nhsuk-grid-column-full">
-        <h1 id="page-heading" class="nhsuk-heading-xl">Centre administrators</h1>
-        <div class="nhsuk-u-margin-top-8">
-          <vc:number-of-administrators centre-id="@Model.CentreId" />
-        </div>
+        <h1 class="nhsuk-heading-xl">Centre administrators</h1>
+
+        <vc:number-of-administrators centre-id="@Model.CentreId" />
       </div>
     </div>
 
-    <form method="get" role="search" asp-action="Index" asp-route-page="@Model.Page">
+    <div class="nhsuk-grid-row">
+      <form method="get" role="search">
+        <input type="hidden" name="filterBy" value="@Model.FilterString" />
+        <vc:search-box label="Search administrators"
+                       asp-controller="Administrator"
+                       asp-action="Index"
+                       searchable-page-view-model="@Model"
+                       css-class="nhsuk-grid-column-one-half" />
+      </form>
+    </div>
 
-      <div class="nhsuk-grid-row">
-        <vc:search-box label="Search administrators" asp-controller="Administrator" asp-action="Index" searchable-page-view-model="@Model" />
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full" id="filters-container">
+        @foreach (var ((filterProperty, filterName), filterValues) in Model.Filters) {
+          <div id="filter-@filterProperty">
+            <form method="get" asp-action="@ViewContext.RouteData.Values["action"].ToString()" asp-route-page="1">

```

Comment:
I assume it was done on the original version of Search/Sort so it would be reusable on multiple pages without having to specify it on view models or wherever. Just keeping the pattern